File: Versão em Python/Algoritmos - Lista/Exercício - 4.py
# Exercício - 4
# Elaborar um programa que calcule e apresente o valor do volume de uma caixa retangular,
# utilizando a fórmula VOLUME <- COMPRIMENTO * LARGURA * ALTURA.

x = input("Digite o valor do Comprimento: ")
print('')
y = input("Digite o valor da Largura: ")
print('')
z = input("Digite o valor da Altura: ")
print('')

comprimento = int(x)
altura = int(y)
largura = int(z)

volume = (comprimento * altura * largura)

print('O volume da caixa retangular é', volume, 'm³')

File: Versão em JavaScript/Algoritmos - Lista/Exercício - 4.js
// Exercício - 4
// Elaborar um programa que calcule e apresente o valor do volume de uma caixa retangular,
// utilizando a fórmula VOLUME <- COMPRIMENTO * LARGURA * ALTURA. 

const prompt = require('prompt-sync')({ sigint: true })

const comprimento = prompt('Digite o comprimento: ')
console.log('=============================================')


const largura = prompt('Digite o valor da largura: ')
console.log('=============================================')


const altura = prompt('Digite o valor da altura: ')
console.log('=============================================')

a = (Number(comprimento))
b = (Number(largura))
c = (Number(altura))

const volume = (comprimento * largura * altura)
console.log(`O volume da caixa retangular é: ${volume}m³`)



